reference/3.0/Microsoft.PowerShell.Core/About/about_Signing.md

```diff
@@ -151,7 +152,7 @@ Microsoft .NET Framework SDK (versions 1.1 and later) and in the Microsoft
 Windows SDK.
 
 For more information about the syntax and the parameter descriptions of the
-MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](http://go.microsoft.com/fwlink/?LinkId=119097)
+MakeCert.exe tool, see [Certificate Creation Tool (MakeCert.exe)](/previous-versions/dotnet/netframework-2.0/bfsktky3(v=vs.80))
 in the MSDN library.
```

Comment:
Remove reference to MSDN library now that the link points directly to Docs.

Reply:
Fixed